lib/jekyll/theme.rb

```diff
@@ -38,7 +38,7 @@ def path_for(folder)
       return unless resolved_dir
 
       path = Jekyll.sanitized_path(root, resolved_dir)
-      path if Dir.exists?(path)
+      path if Dir.exist?(path)
```

Comment:
Use `File.directory?` it's straight to the point.  `Dir.exist?` actually points to `File.directory?`